packages/auth-helpers/src/lib/resources.ts

```diff
@@ -42,6 +45,34 @@ export class LitAccessControlConditionResource
       litAbility === LitAbility.AccessControlConditionSigning
     );
   }
+
+  /**
+   * Composes a resource string by hashing access control conditions and appending a data hash.
+   *
+   * @param {AccessControlConditions} accs - The access control conditions to hash.
+   * @param {string} dataToEncryptHash - The hash of the data to encrypt.
+   * @returns {Promise<string>} The composed resource string in the format 'hashedAccs/dataToEncryptHash'.
+   */
+  public static async composeLitActionResourceString(
```

Comment:
The name is misleading? It's for access control conditions not for Lit Actions.

Reply:
You're right, fixed it here: https://github.com/LIT-Protocol/js-sdk/pull/444/commits/dce95813ca76748925e79a9f52e45853b97e89f4